### PR TITLE
fix(KUXRN-4356): fix for bookmarks actions

### PR DESCRIPTION
**Pull request by Magi-a**
Fixed Bookmarks for ios.

Changed AnalyticsConfig to PhoenixAnalyticsPluginConfig.

### CODE REVIEW

File: ios/KalturaPlayerViewManager.swift [hunk 1]
```diff
@@ -554,7 +554,18 @@ class KalturaPlayerRNView : UIView {
     }
     
     func updateOttAnalyticsConfig(config: Dictionary<String, Any>) {
-        let ottAnalyticsConfig = AnalyticsConfig(params:config)
+        let baseUrl: String = config["baseUrl"] as? String ?? ""
+        
+        let timerInterval: Double = config["timerInterval"] as? Double ?? 0.0
```
Comment (OrenMe) [inline ios/KalturaPlayerViewManager.swift]: What is default in plugin? What do we pass from config? I think 30sec is default

File: ios/KalturaPlayerViewManager.swift [hunk 2]
```diff
@@ -554,7 +554,18 @@ class KalturaPlayerRNView : UIView {
...
+        
+        let ks: String = config["ks"] as? String ?? ""
+        
+        let partnerId: Int = config["partnerId"] as? Int ?? 0
+        
+        let isExperimentalLiveMediaHit: Bool = config["experimentalLiveMediaHit"] as? Bool ?? false
```
Comment (OrenMe) [inline ios/KalturaPlayerViewManager.swift]: What do we pass today from app?

Comment (Magi-a) [reply ios/KalturaPlayerViewManager.swift]: "ottAnalytics": {
			"partnerId": 5002,
			"baseUrl": "https://5002.frp1.ott.kaltura.com/api_v3/",
			"ks": "djJ8NTAwMnzMjM5IyF4euH4PpB36j5_468Hzz3pNp4Mbxh9Cn9w0fWyPaTxCpJuuHjlDPAX27Vpi6jqqVFXWOqJlJBfkejg_zQy9TLbh5AfE2Cixxm_Lv7528-1_vm8gweg2wpWUWhLv7vZsaGKcyQTuLABx6IHpMnmi9ADbhN50rC4HCLvshkleeNUDxuEyuZVrsM14LvknMzofcpv_qaqYm7tAcdz4Csr1pJRKRtxQELjWSfV7rh7gFcKf_IrJV5SQiMNK8FLFL9Xu3SFjZ-ku6HXCdeosOyMPmkK_92HXay5vKZx4Qqom7OX-rNZJSKQgDeCAaF_TW-HKlp55vLUFNm28Ofp_7nXDBAz6Q3BBJyhy9s0kNwes05ltnPsXYzNZ1-Qkl62Oy7J5YjR8PyZrFWYQ8pWhXRnadN6UZ87hrGo6uWntWg==",
			"timerInterval": 30,
			"experimentalLiveMediaHit": true,
			"epgId": 143162515
		}

File: ios/KalturaPlayerViewManager.swift (hunk 1 above)
Comment (Magi-a) [reply ios/KalturaPlayerViewManager.swift]: yes
changed timerInterval => 30
and experimentalLiveMediaHit => true

File: ios/KalturaPlayerViewManager.swift (hunk 2 above)
Comment (Magi-a) [reply ios/KalturaPlayerViewManager.swift]: fixed

**Review by OrenMe — APPROVED** [review]

**Pull request merged**